Podcast/Extensions.swift

```diff
@@ -43,3 +44,18 @@ extension UIFont {
     @nonobjc static let discoverTableViewCellDefaultFontAttributes = [NSForegroundColorAttributeName: UIColor.podcastGrayLight, NSFontAttributeName: UIFont(name: ".SFUIText-Medium", size: 10.0)!]
 
 }
+
+extension CMTime {
+    var durationText:String {
+        let totalSeconds = Int(self.seconds)
+        let hours = totalSeconds / 3600
```

Comment:
need to use floats here, as you're currently dividing two Ints which will give you a rounded Int instead of a double